subsys/net/ip/tcp.c

```diff
@@ -369,15 +369,6 @@ static inline u32_t get_recv_wnd(struct net_tcp *tcp)
 	return min(NET_TCP_MAX_WIN, NET_TCP_BUF_MAX_LEN);
 }
 
-/* True if the (signed!) difference "seq1 - seq2" is positive and less
- * than 2^29.  That is, seq1 is "after" seq2.
- */
-static inline bool seq_greater(u32_t seq1, u32_t seq2)
```

Comment:
Could we also have a helper(s) (like greater and smaller) that returns boolean like this removed function did? It is easier to understand the boolean value than look the function value documentation.